backend/src/controllers/questionController.ts

```diff
@@ -33,3 +33,52 @@ export const createQuestion = async (req: Request, res: Response) => {
     return res.status(400).json({ error });
   }
 };
+
+export const updateQuestion = async (req: Request, res: Response) => {
+  try {
+    const { questionId } = req.params;
+    const validationResult: ValidationResult<CreateQuestionType> =
+      createQuestionValidation.validate(req.body);
+    if (validationResult.error) {
```

Comment:
Can you do another validation for update endpoint as you can update only the title or only the type or option different to creating where the question must have atleast title and type